android/src/main/java/com/zoontek/rnpermissions/RNPermissionsModuleImpl.java

```diff
@@ -47,6 +49,11 @@ public static Map<String, Object> getConstants() {
   }
 
   private static boolean isPermissionUnavailable(@NonNull final String permission) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
```

Comment:
There's no need for this extra check. If the permission is not available in the manifest, it will return `false`